src/mlpack/core/cv/simple_cv_impl.hpp

```diff
@@ -13,6 +13,7 @@
 #define MLPACK_CORE_CV_SIMPLE_CV_IMPL_HPP
 
 #include <cmath>
+#include <utility>
```

Comment:
I think these should be included already by `mlpack/prereqs.hpp`, so this may be superfluous.